WalletWasabi.Gui/Controls/WalletExplorer/TransactionFeeInfo.cs

```diff
@@ -0,0 +1,56 @@
+using NBitcoin;
+using ReactiveUI;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WalletWasabi.Gui.Controls.WalletExplorer
+{
+	public class TransactionFeeInfo : ReactiveObject
```

Comment:
This class is very well welcomed - a good idea. This removes code from the SendTabViewModel. 
The values inside this class should be calculated from the FeeRate property.

Reply:
I think doing the calculations inside of here gets ugly, to do that we need to add the selected coins, `isMax`, and the expected tx size. Seems like a bit much for something holding data for the fee info button.

Reply:
I believe that a simple data container should not do any sophisticated operations. It's typically better to add a class that works on top of data containers.

Reply:
ACK